### DISCUSSION TITLE
Example bug report for bonus

### DISCUSSION THREAD

**Discussion by stan6** [SOEN345-WINTER2024/discussion#3] [Announcements]
### How to reproduce 
1. Lauch memory_game
2. Select theme and difficulty 
3. Click two different icons to flip 
4. Click one of these two icons several times when they flip back 
5. Click another icon, then the icon you clicked before will not flip back
### Actual behavior 
<img src="https://user-images.githubusercontent.com/21846605/50392697-68b43580-078b-11e9-96f6-ba8b8106b5fc.gif" width ="200">

### Expected behavior
<img src="https://user-images.githubusercontent.com/21846605/50392701-7073da00-078b-11e9-81ff-ab3a1bef40d1.gif" width ="200">

App Version: 3.5
Android Version: 13

This issue is reported to [link of the open-source app](https://github.com/Virtuoel/Pehkui/issues/209)

**Comment by marinaGirgis** [comment]
Title: Mindustry App

Platforms
Android

Build
146

Issue
A number of text blocks emerge sequentially at the beginning of the player's first game in Ground Zero on Serpulo (I haven't touched Erekir yet) during the tutorial. Except for one, which is one long line of text with the beginning and conclusion of the phrase extending beyond the width of the phone screen, each block has two to three lines of text. It's likely a minor text formatting error that was overlooked while the game was being resized from a PC with a horizontal screen to a phone with a vertical screen. Although this is a bug complaint for Android, it could be worthwhile to check again to see if the iOS game is experiencing the same problem.
It's at the very start of the game, so you'll be able to find it in less time than it takes to read all this.

Steps to reproduce
Open freshly downloaded virgin app > Campaign > Serpulid > Ground Zero > tap through the start up tutorial until the text tries to give your phone screen a very wide bear hug 

[MindustryAppBug.zip](https://github.com/SOEN345-WINTER2024/discussion/files/14729751/MindustryAppBug.zip)

Duplicate Check
I have searched the existing and closed issues to confirm that this bug has not been reported before.

**Reply by stan6** [reply]
@marinaGirgis hi, can you provide the link of the Github issue where you have filed the bug report? Thank you.

**Comment by dlu2002** [comment]
Here is my issue that I found and posted:

https://github.com/Darkempire78/OpenCalc/issues/448#issue-2244457840
